kotsadm/pkg/store/s3pg/license_store.go

```diff
@@ -85,3 +85,58 @@ func (s S3PGStore) GetLicenseForAppVersion(appID string, sequence int64) (*kotsv
 
 	return nil, nil
 }
+
+func (s S3PGStore) GetAppSlugForLicense(license string) (string, error) {
+	db := persistence.MustGetPGSession()
+	query := `select slug from app where license = $1`
+	row := db.QueryRow(query, license)
+
+	var appSlug string
+	if err := row.Scan(&appSlug); err != nil {
+		if err == sql.ErrNoRows {
+			return "", nil
+		}
+		return "", errors.Wrap(err, "failed to scan")
+	}
+
+	return appSlug, nil
+}
+
+func (s S3PGStore) GetAllAppLicenses() ([]*kotsv1beta1.License, error) {
+	db := persistence.MustGetPGSession()
+	query := `select license from app`
+	rows, err := db.Query(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to query")
+	}
+	defer rows.Close()
+
+	var licenseStr sql.NullString
+	licenses := []*kotsv1beta1.License{}
+	for rows.Next() {
+		if err := rows.Scan(&licenseStr); err != nil {
+			return nil, errors.Wrap(err, "failed to scan")
+		}
+		if licenseStr.Valid {
+			decode := scheme.Codecs.UniversalDeserializer().Decode
+			obj, _, err := decode([]byte(licenseStr.String), nil, nil)
+			if err != nil {
+				return nil, errors.Wrap(err, "failed to decode license yaml")
+			}
+			license := obj.(*kotsv1beta1.License)
+			licenses = append(licenses, license)
+		}
+	}
+
+	return licenses, nil
+}
+
+func (s S3PGStore) GetParsedLicense(licenseStr string) (*kotsv1beta1.License, error) {
```

Comment:
this  function doesn't read anything from the store, i think it should be moved outside the store